vendor/github.com/operator-framework/operator-registry/pkg/containertools/dockerfilegenerator.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 const (
-	defaultBinarySourceImage = "quay.io/operator-framework/upstream-opm-builder"
```

Comment:
This file should not be directly changed. Instead, it seems we need OLM to bump its operator-registry version in `go.mod` and then re-vendor.

Reply:
Probably the re-vendoring would have done the same string replacement. Anyway, I force-pushed that change away, is the vendoring done automatically? I guess yes, as there is a "vendor" step in the workflows